lib/ansible/module_utils/kubevirt.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+#
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
+# (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
+
+from collections import defaultdict
+
+from ansible.module_utils.k8s.raw import KubernetesRawModule
+
+from openshift import watch
+from openshift.helper.exceptions import KubernetesException
+
+API_VERSION = 'kubevirt.io/v1alpha2'
+
+
+VM_COMMON_ARG_SPEC = {
+    'merge_type': {'type': 'list', 'choices': ['json', 'merge', 'strategic-merge']},
+    'wait': {'type': 'bool', 'default': True},
```

Comment:
I think it's dangerous to have 'wait' default to True. Upstream `k8s` has this default to False.

Reply:
It's dangerous in k8s because it's generic and works for everything and they can't wait for everything, those are simplified modules where user would expect to wait by default, because we know what we wait for.

Reply:
I guess the bumped up wait time should be enough for most cases.